middleware/csrf/csrf.go

```diff
@@ -1,10 +1,10 @@
 package csrf
 
 import (
+	"context"
```

Comment:
Maybe, Can we use context from Fiber?

Reply:
I don't think it's necessary, std context should work fine, besides many users will use std context.
If in the future ctx is exposed through config for other reasons that will provide compatibility between libraries